src/com_tjvendors/admin/models/vendors.php

```diff
@@ -208,6 +208,32 @@ public function deleteClientFromVendor($vendor_id,$client)
 		$tjvendorsModelVendor = JModelLegacy::getInstance('Vendor', 'TjvendorsModel');
 		$vendorData           = $tjvendorsModelVendor->getItem($vendor_id);
 		$db                   = $this->getDbo();
+		
+		//Get vendor specific Fees Data
+		$query = $db->getQuery(true);
```

Comment:
Need to use the `getItems()` from the fees model.

Reply:
Resolved